dev-python/pygal/pygal-2.4.0.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+PYTHON_REQ_USE="xml(+)"
+
+inherit distutils-r1
+
+DESCRIPTION="A python SVG charts generator"
+HOMEPAGE="http://pygal.org/"
+# PyPI tarballs do not contain docs
+# https://github.com/Kozea/pygal/pull/428
+SRC_URI="https://github.com/Kozea/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples test"
+
+RDEPEND="
+	dev-python/lxml[${PYTHON_USEDEP}]
+	dev-python/setuptools[${PYTHON_USEDEP}]
```

Comment:
I'm not sure this is needed as a runtime dep.

Reply:
additionally, do you need pytest_runner at install time?

https://github.com/Kozea/pygal/blob/master/setup.py#L77

Reply:
It is, because `pkg_resources` is used by `__init.py__`. pytest-runner would only be used if I ran `setup.py test`.
I chose to run `pytest` on its own because it adds greater flexibility, to exclude tests, for instance.
> Setup scripts can use pytest-runner to add setup.py test support for pytest runner.

Reply:
ya, that works.